app/views/businesses/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if @businesses.count > 0 %>
```

Comment:
I think there is a more active record way to do this, something like if businesses.exist... we can look into it